platform/o.n.swing.plaf/src/org/netbeans/swing/plaf/nimbus/NimbusLFCustoms.java

```diff
@@ -44,8 +44,6 @@ public Object[] createLookAndFeelCustomizationKeysAndValues() {
             fontsize = in.intValue();
         }
 
-        //XXX fetch the custom font size here instead
-        Font controlFont = new Font("Dialog", Font.PLAIN, fontsize); //NOI18N
```

Comment:
this belongs to the commented out section below. Comment it out too or revert the change.